### DISCUSSION TITLE
Create App after argo-cd install using Helm chart ?

### DISCUSSION THREAD

**Discussion by ncasaux** [argoproj/argo-helm#2867] [Q&A]
Hello,

Sorry if it's a silly question, but I'm struggling to find an answer...

I managed to install argo-cd using the Helm chart, and I wanted also to "automatically" configure an App as I would do in argo-cd UI:
![image](https://github.com/user-attachments/assets/218223c3-9ab7-4743-8a81-d67dd23e3805)
Is it possible to configure in some way the values of the chart to create an app ? Is there an example somewhere ?

Thanks in advance!

**Comment by yu-croco** [comment]
Hi @ncasaux , maybe you want to use https://github.com/argoproj/argo-helm/tree/main/charts/argocd-apps ?

**Reply by ncasaux** [reply]
Thank you, I'll have a look at this! :)
